### ISSUE TITLE
Wrong units for the cost?

### ISSUE THREAD

**Issue by alexshturm** [isacssouza/erlgrind#6]
The description says 'costs are in milliseconds', however fprof output is in milliseconds and erlgrind multiplies these values by 1000, so the costs must be in microseconds?

**Comment by Stratus3D** [comment]
Yeah it's got to be microseconds. fprof docs clearly say in milliseconds http://erlang.org/doc/man/fprof.html
